tests/std/tests/P0896R4_ranges_range_machinery/test.cpp

```diff
@@ -34,7 +34,7 @@ STATIC_ASSERT(std::same_as<std::make_unsigned_t<std::ptrdiff_t>, std::size_t>);
 
 // GH-2358: <filesystem>: path's comparison operators are IF-NDR
 static_assert(ranges::range<std::filesystem::path>);
-static_assert(ranges::range<const std::filesystem::path>);
+static_assert(ranges::range<std::filesystem::path const>);

```

Comment:
Why are we standardizing on east const here, instead of the west const that we use throughout the codebase?

Reply:
This is an east const file which has a few inconsistent uses of const west that have snuck in over time. I'm making it locally consistent. I could just as easily make it consistent with the global const west convention, but that would be a larger change.

I think the best thing would be to yank these immaterial west-to-east changes, making this purely a feature change, and submit a separate PR to convert all of the basic ranges machinery (notably including `range_algorithm_support.hpp`) from east-to-west. I'll do that.